src/DotLiquid/Context.cs

```diff
@@ -697,6 +680,12 @@ private static object Liquidize(object obj)
 
         private static bool IsKeyValuePair(object obj)
         {
+            return TryGetKeyValuePair(obj, out var key, out var value);

```

Comment:
```out _``` might be better ? Or just rewrite the only other reference to this function and remove it entirely?

Reply:
Thanks, I haven't seen the discard feature before.